client/components/domains/map-domain-step/index.jsx

```diff
@@ -183,9 +191,23 @@ var MapDomainStep = React.createClass( {
 	},
 
 	handleValidationErrorMessage: function( domain, error ) {
-		let message;
+		let message,
+			errorLevel = 'error';
```

Comment:
Yeah I agree they should've been similar and I think the better one is `severity` since this is not Windows and we are not writing batch files 😁